client/src/main/scala/Board.scala

```diff
@@ -0,0 +1,66 @@
+package client
+
+import game._
+import japgolly.scalajs.react.vdom.prefix_<^._
+import japgolly.scalajs.react.{vdom, ReactComponentB, ReactComponentC}
+import org.scalajs.dom.{raw, svg, html}
+import scalacss.ScalaCssReact._
+
+case class Board(style: BoardStyle) {
+  val component: ReactComponentC.ReqProps[Position[Point19], Unit, Unit, raw.Element] =
+    ReactComponentB[Position[Point19]]("").render_P(p => theBoard(p)).build
+
+  def theBoard(position: Position[Point19]): vdom.ReactTagOf[html.Div] =
+    <.div(style.board,
+      goban,
+      <.div(style.points,
+        Shape[Point19].all.map(position.at).map {
+          case Empty        => <.div(style.empty)
+          case Stone(Black) => <.div(style.black)
+          case Stone(White) => <.div(style.white)
+        }
+      )
+    )
+
+  val goban: vdom.ReactTagOf[svg.SVG] = {
+    import japgolly.scalajs.react.vdom.svg.prefix_<^._
+
+    val u = style.gridUnit
+    val w = 18 * style.gridUnit
+    val c = style.gridUnit / 2.0 + 0.5
+    val gray = "#111"
+
+    val viewBox =
+      ^.viewBox := s"-$c, -$c, ${w + u}, ${w + u}"
+
+    val border = <.rect(
+      ^.height := w,
+      ^.width := w,
+      ^.stroke := gray,
+      ^.strokeWidth := 1,
+      ^.fill := "none")
+
+    val hp = s"m 0 $u h  $w "
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [The string literal  " appears 4 times in the file.](https://www.codacy.com/app/olivier-blanvillain/19x19-net/file/2692327624/issues/source?bid=3301982&fileBranchId=3340022#l43)